src/main/java/models/Suit.java

```diff
@@ -6,6 +6,6 @@
  * Created by michaelhilton on 1/25/16.
  */
 public enum Suit  implements Serializable {
-    Hearts, Spades, Diamonds, Clubs
```

Comment:
I think it's good that we don't have Spanish cards as a separate class but I'm not sure if this follows the design patterns